requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 # Dependencies in Koalas. When you update don't forget to update setup.py and install.rst in docs.
-pandas>=0.23.2,<=1.0.1
+pandas>=0.23.2,<=1.0.*
```

Comment:
I mean, just

```
pandas>=0.23.2
```

should be fine?

Reply:
btw, don't forget to also update `setup.py`.

Reply:
ah, yeah i was a bit confused 😹
And just updated `setup.py` also. thanks!